OPERATOR.rst

```diff
@@ -302,9 +302,14 @@ separately.
 
 There are ways to dynamically determine the latest AMI released by CIS under the
 subscription but in the spirit of reproducible builds, we would rather pin the
-AMI ID and adopt updates at our own discretion to avoid unexpected failures. To
-obtain the latest compatible AMI ID, select the desired ``….gitlab`` component,
-say, ``_select dev.gitlab`` and run
+AMI ID and adopt updates at our own discretion to avoid unexpected failures.
+
+Note that the AMI versioning scheme (e.g., `v01`, `v11`) indicates the month of
+release, and not a sequential release number. For example, `v01` representing
+January of the current year is newer than `v12` (December) of the previous year.
```

Comment:
```suggestion
release, and is not a monotonically increasing value. For example, `v01` representing
January of the current year is newer than `v12` (December) of the previous year.
```

Reply:
```suggestion
release, and not a sequential release number. 
```

The second sentence isn't generally true, a `v01` release may or may not be older than a `v12` release. Then again, this should be immediately apparent to a reasonable person, so I don't think the example is necessary.